packages/ui/src/utils/utils.ts

```diff
@@ -0,0 +1,130 @@
+export const getInitials = (name: string, length?: number) => {
+  // Split the name into an array of words, ignoring empty strings
+  const words = name.split(' ').filter(Boolean)
+
+  // Get the initials from the words
+  const initials = words
+    .map(word => word[0].toUpperCase()) // Get the first letter of each word
+    .join('')
+
+  // If length is provided, truncate the initials to the desired length
+  return length ? initials.slice(0, length) : initials
+}
+export const INITIAL_ZOOM_LEVEL = 1
+export const ZOOM_INC_DEC_LEVEL = 0.1
+
+const LOCALE = Intl.NumberFormat().resolvedOptions?.().locale || 'en-US'
+
+/**
+ * Format a timestamp to medium format date (i.e: Jan 1, 2021)
+ * @param timestamp Timestamp
+ * @param dateStyle Optional DateTimeFormat's `dateStyle` option.
+ */
+export function formatDate(timestamp: number | string, dateStyle = 'medium'): string {
+  return timestamp
+    ? new Intl.DateTimeFormat(LOCALE, {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore: TS built-in type for DateTimeFormat is not correct
```

Comment:
TS's built-in style does work, but most people forget that it's part of the Intl package. This should work...
```suggestion
export function formatDate(timestamp: number | string, dateStyle: Intl.DateTimeFormatOptions['dateStyle'] = 'medium'): string {
  return timestamp
    ? new Intl.DateTimeFormat(LOCALE, {
```